Doc/tutorial/inputoutput.rst

```diff
@@ -258,9 +258,9 @@ Old string formatting
 ---------------------
 
 The ``%`` operator can also be used for string formatting. It interprets the
-left argument much like a :c:func:`sprintf`\ -style format string to be applied
-to the right argument, and returns the string resulting from this formatting
-operation. For example::
+left argument much like to C's :c:func:`sprintf`\ -style format string to be
```

Comment:
I'm accord this a lot

> this should aim to be a new-user friendly as possible and not expect prior knowledge.

What if we removed the sprintf mention? Something like this:

`The % operator (modulo) can also be used for string formatting. Given ``string % values``, % conversion specifications in ``string`` are replaced with zero or more elements of ``values``. This is also known as interpolation operator. For example:`

Additionally, this section already has a link to https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting.

Reply:
@adorilson:
> What if we removed the sprintf mention? Something like this:

Hmm, this is an interesting idea. String interpolation is a far more universal concept across several languages, and is likely to be more relatable to users of other languages without requiring the prior knowledge. 

My suggestion would be to change:
```
This is also known as interpolation operator.
```
To this:
```
This operation is commonly known as string interpolation.
```
While "interpolation operator" is not incorrect, I've seen "string interpolation" mentioned far more commonly. See https://en.wikipedia.org/wiki/String_interpolation and https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated. In any search engine, using the query ``string interpolation <language>`` is pretty much guaranteed to find the correct results for any widely used programming language.

Also as a more minor suggestion, I would recommend changing:
```
Given ``string % values``, % conversion specifications in ``string`` are replaced with zero or more elements of ``values``.
```
To this:
```
Given ``'string' % values``, instances of ``%`` in ``string`` are replaced with zero or more elements of ``values``.
```
I think this phrasing make a bit more sense to readers and adds a bit of clarity with the formatting.

Good idea overall though, I'm definitely on board with it myself. +1

Reply:
Great. I will do this. But, I have a question. In my local branch, Do I must revert this commit or just add a new?

Reply:
@adorilson:
> In my local branch, Do I must revert this commit or just add a new?

Either way works. Since this will all get squashed into a single commit when/if the PR is merged, the actual commit log within the PR branch doesn't matter much. I usually prefer to add commits so that I can revert back to it at a later point in time if needed, but it's up to personal preference.

Reply:
@aeros167 It's done. Thanks for suggestions.